__tests__/index.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
@mattiaerre Any specific reason why we re renaming the test dir into `__tests__`?